### ISSUE TITLE
Clicking a carousel-indicator causes page to scroll

### ISSUE THREAD

**Issue by gabrieltaets** [gbrlsnchs/material2-carousel#33]
**Conditions:**
1. A carousel is partially hidden due to page scroll position, but the indicators are still visible.
2. User clicks on an indicator.

**Expected:**
- Carousel jumps to the clicked slide, no other effects.

**Actual:**
- Carousel jumps to the clicked slide, but also forces a page scroll to the top of the carousel

**Comment by gbrlsnchs** [comment]
Yeah, this is intended behavior, however it sucks on mobile because of that. For now, a fast solution would be to enable turning it off.
